package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "openhim-core",
   "description": "The OpenHIM core application that provides logging and routing of http requests",
-  "version": "8.4.2",
+  "version": "8.5.0",
```

Comment:
I think it might actually be a breaking change as the old way permissions were set for users won't work anymore.